dags/addons_daily.py

```diff
@@ -0,0 +1,41 @@
+from airflow import DAG
+from airflow.operators.sensors import ExternalTaskSensor
+from datetime import datetime, timedelta
+from operators.emr_spark_operator import EMRSparkOperator
+#from utils.constants import DS_WEEKLY
+
+default_args = {
+    'owner': 'bmiroglio@mozilla.com',
+    'depends_on_past': True,
+    'start_date': datetime(2019, 5, 15),
+    'email': ['telemetry-alerts@mozilla.com', 'bmiroglio@mozilla.com',
+    'smelancon@mozilla.com','bwright@mozilla.com','dthorn@mozilla.com'],
+    'email_on_failure': True,
+    'email_on_retry': True,
+    'retries': 2,
+    'retry_delay': timedelta(minutes=30),
+}
+
+dag = DAG('addons_daily', default_args=default_args, schedule_interval='@daily')
+
+wait_for_main_summary = ExternalTaskSensor(
+    task_id='wait_for_main_summary',
+    external_dag_id='main_summary',
+    external_task_id='main_summary',
+    execution_delta=timedelta(hours=-1), # main_summary waits one hour, execution date is beginning of the week
+    dag=dag)
+
+usage_report = EMRSparkOperator(
```

Comment:
Drive by comment: It's preferable to use the `MozDatabricksOperator` over the `EMRSparkOperator` since it includes python 3 support, among other things. This standalone [script](https://raw.githubusercontent.com/mozilla/python_mozetl/master/bin/mozetl-databricks.py) should include most of the features of the operator. One modification that would have to be made is to expose a cli module so `python -m addons_daily.cli` is valid.

After generating a databricks token in the user settings page, the job could be tested as follows on a cluster. 
```
python mozetl-databricks.py \
    --git-path https://github.com/mozilla/addons_daily.git \
    --git-branch master \
    --module-name addons_daily \
    --token <TOKEN>  \
    addons_report \
        --date 20190601
```

Here's another job that's scheduled as a standalone module in a similar way: https://github.com/mozilla/telemetry-airflow/blob/e989db0f60c50e27b0a496978420352ae3c7219e/dags/bgbb.py#L32-L56